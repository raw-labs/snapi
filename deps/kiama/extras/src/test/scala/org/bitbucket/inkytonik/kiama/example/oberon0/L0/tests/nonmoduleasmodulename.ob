MODULE aname;

VAR
    i : INTEGER;

END i.
