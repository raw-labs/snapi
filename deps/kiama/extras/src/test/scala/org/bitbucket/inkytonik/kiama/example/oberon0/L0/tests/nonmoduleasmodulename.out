* _pp.ob
MODULE aname;

    VAR
        i : INTEGER;

END i.
* errors
src/test/scala/org/bitbucket/inkytonik/kiama/example/oberon0/L0/tests/nonmoduleasmodulename.ob:6:5:error: end module name 'i' should be 'aname'
END i.
    ^
